main.tex

```diff
@@ -113,40 +113,40 @@ \section{Introduction} \label{sec:intro}
-might be performed in the course of astronomical research, for example, interfacing with 
+in the quantitative sciences, where researchers must simultaneously produce research, perform
+data analysis, and develop software. A large part of this success owes itself to the vibrant
+community of developers and a continuously-growing ecosystem of tools, web services, and stable
+well-developed packages that enable easier collaboration on software development, easier
+writing and sharing of software documentation, and continuous testing and validation of
+software. While the programming language provides support for array representation and
+arithmetic \citep[\package{numpy};][]{numpy}, a wide variety of functions for scientific
+computing \citep[\package{scipy};][]{scipy}, and publication-quality plotting
+\citep[\package{matplotlib};][]{matplotlib}, tens of thousands of other high-quality and easy-to-use
+packages are available, which can help with tasks that are not astronomy specific but
+might be performed in the course of astronomical research; e.g., interfacing with
 databases or statistical inferences. More recently, the development of package managers such as
 Anaconda\footnote{\url{https://anaconda.org/}} have streamlined the installation process for
-most packages significantly reducing the barrier to entry for using many such libraries.
+most packages, significantly reducing the entry barrier for using many such libraries.
 
-The \astropy project aims to provide an open-source, open-development core
+The \astropy project aims to provide an open-source and open-development core
 package (\astropypkg) and an ecosystem of \emph{affiliated packages} that
 support astronomical functionality in the \python programming language.
 The \astropypkg core package is now a feature-rich library of sufficiently
 general tools and classes that supports the development of more specialized
-code. An example of such functionality is reading and writing FITS files: it would be
+code. An example of such functionality is reading and writing FITS files: It would be
 time consuming and impractical for multiple groups to implement the FITS
 standard \citep{FITS} and maintain software for such a general-purpose need.
 Another example of such a common task is in dealing with representations of and
 transformations between astronomical coordinate systems.
 
-The \astropy project aims to develop and provide high quality code and
+The \astropy project aims to develop and provide high-quality code and
 documentation according to the best practices in software development.
 The project makes use of these tools to do so without central institutional
 oversight.
 The first public release of the \astropypkg package is described in
 \cite{astropy}. Since then, the \astropypkg package has been
-used in hundreds of projects, and the scope of the package has grown
-considerably. At the same time, the community of astronomers
```

Comment:
NOTE: Not all contributors are strictly astronomers.

Reply:
very much agree with this, thanks!